api/v1beta2/integrationtestscenario_webhook.go

```diff
@@ -53,6 +53,38 @@ func (r *IntegrationTestScenario) ValidateCreate() (warnings admission.Warnings,
 					"param manually defined because it will be automatically generated"+
 					"by the integration service")
 		}
+		// we won't enable ITS if git resolver with url & repo+org
+		urlResolverExist := false
+		repoResolverExist := false
+		orgResolverExist := false
+
+		for _, gitResolverParam := range r.Spec.ResolverRef.Params {
+			if gitResolverParam.Name == "url" && gitResolverParam.Value != "" {
+				urlResolverExist = true
+			}
+			if gitResolverParam.Name == "repo" && gitResolverParam.Value != "" {
+				repoResolverExist = true
+			}
+			if gitResolverParam.Name == "org" && gitResolverParam.Value != "" {
+				orgResolverExist = true
+			}
+		}
+
+		if urlResolverExist {
+			if repoResolverExist || orgResolverExist {
+				return nil, field.Invalid(field.NewPath("Spec").Child("ResolverRef").Child("Params"), param.Name,
+					"an IntegrationTestScenario resource should not have both gitResolver "+
```

Comment:
this sentence looks unfinished

Reply:
do you mean the sentence is not correct ? 

```
"an IntegrationTestScenario resource should not have both gitResolver "+
						"Either url, or repo (with org) must be specified, but not both.")
```

Reply:
`Either` starts with capital letter, it means new sentence. While there is no full stop `.` in previous line.

`both gitResolver` both what? Thee should be 2 things named.